notebooks/CV-data-loading-benchmarking.ipynb

```diff
@@ -246,7 +246,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.12"
+   "version": "3.10.9"
```

Comment:
Looks unnecessary to update the python version.

Reply:
lol always updated... kind of annoying...